estate/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+access_estate_property,access_estate_property,model_estate_property,base.group_user,1,1,1,1
```

Comment:
I know runbot is not red, but it's better to always add a line break at the end of a file (_again to minimize the GitHub diff when someone later adds a line to your file_)